CHANGELOG.md

```diff
@@ -137,6 +137,8 @@ If you are looking for developer-facing changes, check out [CHANGELOG-API.md](./
 - `signaltometricsconnector`: Fix same metric name, unit, and type with different description ignoring description (#39736)
 - `receiver/sqlserver`: Properly parse numbers stored in scientific notation to integers (#39124)
 
+- `exporter/elasticsearch`: Fix panic when encoding non-string scope attributes #40098
```

Comment:
Add a file to .chloggen instead. See https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/CONTRIBUTING.md#changelog

Reply:
@carsonip I fixed the changelog.